pkg/scheduler/framework/plugins/volumezone/volume_zone_test.go

```diff
@@ -229,6 +229,18 @@ func TestSingleZone(t *testing.T) {
 			},
 			wantFilterStatus: framework.NewStatus(framework.UnschedulableAndUnresolvable, ErrReasonConflict),
 		},
+		{
+			name: "pv with beta label,node with ga label",
```

Comment:
One thing I'm not sure is: in this pr, we let pv with beta label to be scheduled to node with ga label.
Should we make this work the other way around as well? That is,  pv with ga label to be scheduled to node with beta label.

Reply:
I think it is fine to leave out. We converted PVs to use the GA label after introducing the GA label on Nodes.